arkouda/numeric.py

```diff
@@ -422,9 +425,19 @@ def cos(pda: pdarray) -> pdarray:
     return create_pdarray(repMsg)
 
 
+def _hash_helper(a):
+    from arkouda import Categorical as _Categorical
+
+    if isinstance(a, _Categorical):
+        return a.categories[a.codes]
```

Comment:
we should update how we handle cats to hash the categories and then do integer indexing instead

Reply:
I think we should handle adding categoricals in a separate issue too, since we really want to do this idexing and hashing in the chapel code (so it can be easily called by `hashArrays`